src/services/cube-handler.ts

```diff
@@ -1354,6 +1521,20 @@ async function dateDimensionProcessor(
     //   ?.push(
     //     pgformat("strftime(%I.end_date, '%d/%m/%Y') AS %I", dimTable, t('column_headers.end_date', { lng: locale }))
     //   );
+    defaultSortSelectStatementsMap.get(locale)?.push(pgformat('%I.description AS %I', dimTable, columnName));
+    defaultSortSelectStatementsMap.get(locale)?.push(pgformat('%I.end_date AS %I', dimTable, `${columnName}_sort`));
+    rawSortSelectStatementsMap.get(locale)?.push(pgformat('%I.description AS %I', dimTable, columnName));
+    rawSortSelectStatementsMap.get(locale)?.push(pgformat('%I.end_date AS %I', dimTable, `${columnName}_sort`));
+    // rawSelectStatementsMap
+    //   .get(locale)
+    //   ?.push(
+    //     pgformat("strftime(%I.start_date, '%d/%m/%Y') AS %I", dimTable, t('column_headers.start_date', { lng: locale }))
+    //   );
+    // rawSelectStatementsMap
+    //   .get(locale)
+    //   ?.push(
+    //     pgformat("strftime(%I.end_date, '%d/%m/%Y') AS %I", dimTable, t('column_headers.end_date', { lng: locale }))
+    //   );
```

Comment:
More commented code, is this staying in?

Reply:
See above